dist/src/main/resources/logging/StackdriverLayout.json

```diff
@@ -0,0 +1,80 @@
+{
+  "timestampSeconds": {
+    "$resolver": "timestamp",
+    "epoch": {
+      "unit": "secs",
+      "rounded": true
+    }
+  },
+  "timestampNanos": {
+    "$resolver": "timestamp",
+    "epoch": {
+      "unit": "secs.nanos"
+    }
+  },
+  "severity": {
+    "$resolver": "pattern",
+    "pattern": "%level{OFF=DEFAULT, ALL=DEFAULT, TRACE=DEBUG, DEBUG=DEBUG, INFO=INFO, WARN=WARNING, ERROR=ERROR, FATAL=CRITICAL}",
+    "stackTraceEnabled": false
+  },
+  "message": {
+    "$resolver": "pattern",
+    "pattern": "%m",
+    "stackTraceEnabled": false
+  },
+  "logging.googleapis.com/sourceLocation": {
+    "file": {
+      "$resolver": "source",
+      "field": "fileName"
+    },
+    "line": {
+      "$resolver": "source",
+      "field": "lineNumber"
+    },
+    "function": {
+      "$resolver": "pattern",
+      "pattern": "%replace{%C.%M}{^\\?\\.$}{}",
+      "stackTraceEnabled": false
+    }
+  },
+  "contextMap": {
```

Comment:
💭 The standard template for GCP puts MDC under `logging.googleapis.com/labels`: https://github.com/apache/logging-log4j2/blob/2.x/log4j-layout-template-json/src/main/resources/GcpLayout.json

That would allow us to create some metrics based on logs: https://cloud.google.com/logging/docs/logs-based-metrics/labels

I think that's pretty neat, maybe we want to do the same?